sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/Jsr305AnnotationsCheck.java

```diff
@@ -0,0 +1,996 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2019 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.github.sevntu.checkstyle.checks.coding;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import antlr.collections.AST;
+import com.github.sevntu.checkstyle.SevntuUtil;
+import com.puppycrawl.tools.checkstyle.api.AbstractCheck;
+import com.puppycrawl.tools.checkstyle.api.DetailAST;
+import com.puppycrawl.tools.checkstyle.api.FullIdent;
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
+
+/**
+ * Jsr305AnnotationsCheck - a check to enforce use of nullness anotations.
+ * The standalone version of this plugin resides under
+ * <a href="https://github.com/bjrke/JSR305CheckstylePlugin">
+ * https://github.com/mbert/JSR305CheckstylePlugin</a>
+ * <p>
+ * Copyright (C) 2008 Marcus Thiesen (initial version) Copyright (C) 2008-2019 Jan Burkhardt
+ * (maintainer)
+ * <br>
+ * Thanks to Mattias Nissler, Thorsten Ehlers, Fabian Loewner, Ole Langbehn for contributions.
```

Comment:
please remove this.
We can move this to some project Readme file if really want names to bee present somewhere.

Reply:
That's a duplicate, right?

Reply:
> We can move this to some project Readme file if really want names to bee present somewhere.

I think that would be nice. Would README.textile be an appropriate location? Or should a separate file be added?

Reply:
In readme at root of project, we have some list of people contributed to project, very outdated list, but if want, you can extend it.

Reply:
OK, will do. Since I am dealing with other people's code here (the original authors of that plugin who kindly agreed do have it integrated here) I don't want to offend anybody.

Reply:
We usually do not reference all contributors(initial author, bug fixing, reviewers, .....) in code as there are too much of them and becomes madness. We respect the most user who actually did most critical work to got a merge approval, so it is referenced as "Contributor" in github UI.

Reply:
Yes, I fully understand. Since I am bringing in code from my former colleagues I wanted to make sure they're mentioned somewhere. I have moved this into `README.textile`, please take a look at the next iteration.